### PR TITLE
create Github actions

### PR DESCRIPTION
**Pull request by arthur-cw**
improving the repo

### CODE REVIEW

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'npm'
+    - run: npm install
+    - run: npm install hardhat@2.9.9
```
Comment (EvgeniiZaitsevCW) [inline .github/workflows/node.js.yml:30]: This line seems redundant. Some Hardhat version has already been mentioned in the [package.json](https://github.com/cloudwalk/brlc-token/blob/main/package.json) file. Or we need a concrete version of Hardhat to perform GitHub actions?

File: .github/workflows/node.js.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+name: Node.js CI
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build:
```
Comment (EvgeniiZaitsevCW) [inline .github/workflows/node.js.yml:13]: Renaming suggestion for the job name: `build` => `compile-contracts`

**Review by EvgeniiZaitsevCW — CHANGES_REQUESTED** [review]
Good idea in general!
Then we can extend it with a job to run tests.
Only two small remarks

**Comment by arthur-cw** [comment]
yeah, I agree @EvgeniiZaitsevCW. Do you know how long the tests are taking on avg?

File: .github/workflows/node.js.yml (hunk 1 above)
Comment (arthur-cw) [reply .github/workflows/node.js.yml:30]: I added it because I had to do it locally for some reason. It's safe to remove.

**Review by EvgeniiZaitsevCW — APPROVED** [review]

**Comment by EvgeniiZaitsevCW** [comment]
@arthur-cw 
> Do you know how long the tests are taking on avg?
On my machine running all the tests (276 pcs.) from the `token-bridge` branch is taken about 45 seconds.

**Review by EvgeniiZaitsevCW — APPROVED** [review]
Looks nice!
It would be nice to break some test (or add a non-passing one) and check that the appropriate job failed :)

**Pull request merged**